docs/worker-node/install-singularity.md

```diff
@@ -151,6 +151,7 @@ unprivileged.
     The first option enables unshare to be called (which is needed to
```

Comment:
These two suggestions leave it inconsistent, because there's another mention of "the second option" in the first sentence; I think it looks bad to only say "the second option" without ever mentioning the first option.  On the other hand I think replacing all 3 cases make it unnecessarily long.  I prefer it the way it is overall, but if you feel strongly that all 3 cases should be replaced I won't object.

Reply:
Yeah, I'd prefer direct references to the options in the prose so it'd be great if you could make those changes!